app/controllers/samples_controller.rb

```diff
@@ -189,6 +190,7 @@ def metadata
       ercc_comparison = pr.compare_ercc_counts
 
       job_stats_hash = job_stats_get(pr.id)
+      assembled_taxids = pr.assembled_taxids ? JSON.parse(pr.assembled_taxids) : []
```

Comment:
I think you can put assembled_taxids = [] up top in case pr is nil

Reply:
Another way of writing it that's cute is `JSON.parse(pr.assembled_taxids || "[]")`

Reply:
Thanks!